packages/aws-cdk-lib/aws-apigatewayv2/lib/websocket/integration-response.ts

```diff
@@ -0,0 +1,193 @@
+import { Construct } from 'constructs';
+import { ContentHandling, WebSocketRouteIntegration } from './integration';
+import { IResource, Resource } from '../../../core';
+import { CfnIntegrationResponse } from '../apigatewayv2.generated';
+
+/**
+ * WebSocket integration response key helper class
+ */
+export class WebSocketIntegrationResponseKey {
+  /**
+   * Match all responses
+   */
+  public static default = new WebSocketIntegrationResponseKey('$default');
+
+  /**
+   * Match all 2xx responses (HTTP success codes)
+   */
+  public static success = WebSocketIntegrationResponseKey.fromStatusRegExp(/2\d{2}/.source);
+
+  /**
+   * Match all 4xx responses (HTTP client error codes)
+   */
+  public static clientError = WebSocketIntegrationResponseKey.fromStatusRegExp(/4\d{2}/.source);
+
+  /**
+   * Match all 5xx responses (HTTP server error codes)
+   */
+  public static serverError = WebSocketIntegrationResponseKey.fromStatusRegExp(/5\d{2}/.source);
+
+  /**
+   * Generate an integration response key from an HTTP status code
+   *
+   * @example
+   * // Match 403 status code
+   * apigwv2.WebSocketIntegrationResponseKey.fromStatusCode(403)
+   *
+   * @param httpStatusCode HTTP status code of the mapped response
+   */
+  public static fromStatusCode(httpStatusCode: number): WebSocketIntegrationResponseKey {
+    return new WebSocketIntegrationResponseKey(`/${httpStatusCode}/`);
+
+  }
+
+  /**
+   * Generate an integration response key from a regular expression matching HTTP status codes
+   *
+   * @example
+   * // Match all 20x status codes
+   * apigwv2.WebSocketIntegrationResponseKey.fromStatusRegExp('20\\d')
```

Comment:
I wanted to give an alternative to `WebSocketIntegrationResponseKey.success`, since it already offers `2xx`:

https://github.com/nmussy/aws-cdk/blob/d4c87f2c90a3642d6c50877bf29ddc2c1085e295/packages/aws-cdk-lib/aws-apigatewayv2/lib/websocket/integration-response.ts#L15-L18

But you're right, it can lead to confusion, I'll come up with another example

Reply:
I've updated (and fixed) the example